bonita/content.md

```diff
@@ -14,397 +14,281 @@ Bonita (called Bonita BPM till 7.5) is an open-source business process managemen
 $ docker run --name bonita -d -p 8080:8080 %%IMAGE%%
 ```
 
-This will start a container running [Bonita runtime](https://documentation.bonitasoft.com/bonita/2021.2/tomcat-bundle): a Tomcat bundle with Bonita Engine + Bonita Portal. With no environment variables specified, it's as if you have launched the bundle on your host using startup.{sh|bat} (with security hardening on REST and HTTP APIs, cf Security part). Bonita uses a H2 database here.
+This will start a container running [Bonita runtime](https://documentation.bonitasoft.com/bonita/2022.1/tomcat-bundle): a Tomcat bundle with Bonita Engine + Bonita Portal. With no environment variables specified, it's as if you have launched the bundle on your host using startup.{sh|bat} (with security hardening on REST and HTTP APIs, cf Security part). Bonita uses a H2 database here.
```

Comment:
```suggestion
This will start a container running [Bonita runtime](https://documentation.bonitasoft.com/bonita/latest/tomcat-bundle): a Tomcat bundle with Bonita Engine + Bonita Portal. With no environment variables specified, it's as if you have launched the bundle on your host using startup.{sh|bat} (with security hardening on REST and HTTP APIs, cf Security part). Bonita uses a H2 database here.
```